docs/src/man/gggenome.md

```diff
@@ -0,0 +1,153 @@
+# name  strand  start   end     snippet snippet_pos     snippet_end
+chr1    +       19461997        19462014        AGTTATTCAGCTTTCTATCACGATCAGAGAACAAGCTGAGAAAAGGATGTTTTTGCTTTTGCTTTTGTTTTTCTTCTTATTTTGGAGTTCTCATCCATGATTCATTGACACCATTGCTTTGGCCTCTGGGAAGGGCAGCATATCTGGGTAAAAGCAGATAGCAGAGCAAATCTGCTTACTGCAACCAGCCAGGAAGGAAGCAATGAAAGCACGTTCAC  19461897        19462114
+chr1    +       98281503        98281520        TCTAGTGAGGAGAAATGTAAGCTAACGTGATAAACATTGTTTCTGATACACTAATTAAACTGACTTTTGAAAAGATGGCTTACATGTCTATCTAACATGTTTCATTGACACCATTGCTATAGTATGTAATTTTAATGTAAAATAGCCTTCTTTGCAGGGAATCCAGCCTGCTGCTGAATCTTTAAATTTTCAGTGTCTGTTGTCATAGTAACCAGAAT  98281403        98281620
+...
+```
+
+
+### Understanding `output` parameters
+
+By default, `gggenome()` returns a HTTP.Messages.Response object.
+
+```
+julia> query = "GTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGA";
+
+julia> res = gggenome(query=query, db="refseq", format="txt");
+
+julia> typeof(res)
+HTTP.Messages.Response
+```
+
+By setting `output="toString"`, `gggenome()` returns a String object.
+
+
+```
+julia> query = "GTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGA";
+
+julia> res = gggenome(query=query, db="refseq", format="txt", output="toString");
+
+julia> typeof(res)
+String
+
+julia> println(res)
+# [ GGGenome | 2018-07-01 22:25:16 ]
+# database:     RefSeq complete RNA release 88 (May, 2018)
+# query:        GTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGA
+# count:        15
+# query:        TCCCGCCGGCTTCTCCGGGATCGGTCGCGTTACCGCAC
+# count:        10
+# name  strand  start   end     snippet snippet_pos     snippet_end
+NR_003279.1 Mus musculus 28S ribosomal RNA (Rn28s1), ribosomal RNA      +       2326    2363    GAAGGGACGGGCGATGGCCTCCGTTGCCCTCGGCCGATCGAAAGGGAGTCGGGTTCAGATCCCCGAATCCGGAGTGGCGGAGATGGGCGCCGCGAGGCCAGTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGAGGCCTCGGGGAGAGTTCTCTTTTCTTTGTGAAGGGCAGGGCGCCCTGGAATGGGTTCGCCCCGAGAGAGGGGCCCGTGCCTTGGAAAGCGTCGCGGTTCC      2226    2463
+NR_003287.4 Homo sapiens RNA, 28S ribosomal N5 (RNA28SN5), ribosomal RNA        +       2574    2611    GGGACGGGCGATGGCCTCCGTTGCCCTCGGCCGATCGAAAGGGAGTCGGGTTCAGATCCCCGAATCCGGAGTGGCGGAGATGGGCGCCGCGAGGCGTCCAGTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGAGCCCCGGGGAGAGTTCTCTTTTCTTTGTGAAGGGCAGGGCGCCCTGGAATGGGTTCGCCCCGAGAGAGGGGCCCGTGCCTTGGAAAGCGTCGCGGTTCCG      2474    2711
+...
+```
+
+By setting `output="extractTopHit"`, `gggenome()` returns a String object containing the top hit (Currently, only works with `format="txt"`).
+
+```
+julia> query = "GTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGA";
+
+julia> res = gggenome(query=query, db="refseq", format="txt", output="extractTopHit");
+
+julia> typeof(res)
+String
+
+julia> println(res)
+NR_003279.1 Mus musculus 28S ribosomal RNA (Rn28s1), ribosomal RNA      +       2326    2363    GAAGGGACGGGCGATGGCCTCCGTTGCCCTCGGCCGATCGAAAGGGAGTCGGGTTCAGATCCCCGAATCCGGAGTGGCGGAGATGGGCGCCGCGAGGCCAGTGCGGTAACGCGACCGATCCCGGAGAAGCCGGCGGGAGGCCTCGGGGAGAGTTCTCTTTTCTTTGTGAAGGGCAGGGCGCCCTGGAATGGGTTCGCCCCGAGAGAGGGGCCCGTGCCTTGGAAAGCGTCGCGGTTCC      2226    2463
+```
+
+
```

Comment:
You also need to add an entry to docs/mkdocs.yml and docs/src/index.md to add the page for GGGenome